game-core/src/main/java/games/strategy/engine/data/properties/ComboProperty.java

```diff
@@ -62,4 +60,8 @@ public JComponent getEditorComponent() {
   public boolean validate(final Object value) {
     return possibleValues != null && possibleValues.contains(value);
   }
+
+  public Collection<T> getPossibleValues() {
+    return new ArrayList<>(possibleValues);
+  }
```

Comment:
I'd actually prefer a write protected variant instead of copying the entire list, ditto for the 2 hash maps below

Reply:
`ComboProperty` changed to use `unmodifiableCollection()` in #4645.

The maps in `GameProperties` are mutable.  To avoid the issues discussed previously regarding `ConcurrentModificationException`, I'd prefer leaving the defensive copies in place rather than trying to predict what a caller may or may not do with the returned collections (e.g. storing them, passing them to another receiver in a different thread, etc.).